dydx/dydxPresenters/dydxPresenters/_v4/MarketInfo/Components/dydxMarketPositionViewPresenter.swift

```diff
@@ -88,14 +86,14 @@ class dydxMarketPositionViewPresenter: HostedViewPresenter<dydxMarketPositionVie
         viewModel?.closePrice = dydxFormatter.shared.dollar(number: position.exitPrice?.doubleValue, digits: configs.displayTickSizeDecimals?.intValue ?? 0)
 
         let fundingSign: PlatformUISign
-        if position.netFunding?.doubleValue ?? 0 > 0 {
+        let funding = dydxFormatter.shared.dollarVolume(number: abs(position.netFunding?.doubleValue ?? 0), digits: 2)
+        if formattedZero == funding {
+            fundingSign = .none
+        } else if position.netFunding?.doubleValue ?? 0 > 0 {
             fundingSign = .plus
-        } else if position.netFunding?.doubleValue ?? 0 < 0 {
-            fundingSign = .minus
         } else {
-            fundingSign = .none
+            fundingSign = .minus
```

Comment:
I had to change this logic around so that values like -0.0001 do not show as "-$0.00"

The first check is the 0 check.

Woud you rather I move this logic to the `dollarVolume` function?

Reply:
@johnqh actually since we use a `SignedAmountViewModel`, the `fundingSign` forces a +/- sign so we have to do the `if formattedZero == funding {` check here.

The other option would be to introduce a param `shouldDisplayPlusSignForPositiveNumbers` to the `dollarVolume` function and get rid of the `SignedAmountViewModel`. I kind of prefer that

e.g.

Reply:
@johnqh this logic is now consolidated in the [SignedAmountViewModel extension init](https://github.com/dydxprotocol/v4-native-ios/pull/86/files#diff-13f14d9648251ad010a8e200b10ff2b39e9d9920b14f6754c94f6fdeb9599dbd)